resources/kiali/values.yaml

```diff
@@ -1,82 +1,699 @@
+
 tests:
   enabled: true
 
-server:
-  # The port that the server will bind to in order to receive console and API requests.
-  port: 20001
-  metrics:
-    # When true, the metrics endpoint will be available for Prometheus to scrape.
-    enabled: true
-    # The port that the server will bind to in order to receive metric requests.
-    # This is the port Prometheus will need to scrape when collecting metrics from Kiali.
-    port: 9090
-  # Defines the context root path for the Kiali console and API endpoints and readiness probes.
-  # When providing a context root path that is not "/", do not add a trailing slash.
-  # For example, use "/kiali" not "/kiali/".
-  webRoot: /
+global:
+  isLocalEnv: false
+  istio:
+    gateway:
+      name: kyma-gateway
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
+  alpine_net:
+    name: alpine-net
+    dir:
+    version: "2fbe4fd3"
 
-deployment:
-  image:
-    repository: eu.gcr.io/kyma-project/incubator/develop/kiali
-    tag: 1.14.0-6f374cc6
+kcproxy:
   replicaCount: 1
-  # When true, Kiali will be in "view only" mode, allowing the user to view and retrieve
-  # management and monitoring data for the service mesh, but not allow the user to
-  # modify the service mesh.
-  viewOnlyMode: true
-  # A list of namespaces Kiali is to be given access to.
-  # These namespaces have service mesh components that are to be observed by Kiali.
-  # You can provide names using regex expressions matched against all namespaces the operator can see.
-  # The default makes all namespaces accessible except for some internal namespaces that typically should be ignored.
-  # NOTE! If this has an entry with the special value of "**" (two asterisks), that will denote you want
-  # Kiali to be given access to all namespaces via a single cluster role (if using this special value of "**",
-  # you are required to have already granted the operator permissions to create cluster roles and cluster role bindings).
-  accessibleNamespaces: "**"
+  inPort: 3000
+  clientId: kiali
+  clientSecret: hiFWLWqIxw5d3gl
```

Comment:
Is it really ok to have this information public?

Reply:
They are introduced in the dex config, also for other clients like console.
I found no better way of re-using the setting so that it could be also just generated in the dex config and referenced here.
So I at least made it configurable so that an admin could override both places with custom values.
It was like that before... Open for finding a better solution in a different PR solving the problem for all clients consistently.